common/testing/testvars/test_vars.go

```diff
@@ -171,9 +171,9 @@ func (tv *TestVars) WorkflowExecution(key ...string) *commonpb.WorkflowExecution
 	}
 }
 
-func (tv *TestVars) UpdateRef() *updatepb.UpdateRef {
+func (tv *TestVars) UpdateRef(key ...string) *updatepb.UpdateRef {
```

Comment:
Why is this key optional? How am I supposed to know how many elements this expects?

I generally don't see the value of this whole utility struct but 🤷.